compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -2501,6 +2506,27 @@ static LogicalResult setRootConfig(mlir::FunctionOpInterface entryPointFn,
       DispatchLoweringPassPipeline::CPUDoubleTilingExpert);
 }
 
+static LogicalResult setRootConfig(mlir::FunctionOpInterface entryPointFn,
+                                   IREE::LinalgExt::TopkOp topkOp) {
+  unsigned typeWidthInBytes = IREE::Util::getRoundedElementByteWidth(
+      topkOp.getInputType().getElementType());
+  int64_t typeVectorSize = getVectorSize(entryPointFn, typeWidthInBytes);
+  SmallVector<int64_t> vecTileSizes{0, typeVectorSize};
```

Comment:
Use assignment style to avoid potential bugs and for consistency.

```suggestion
  SmallVector<int64_t> vecTileSizes = {0, typeVectorSize};
```

https://abseil.io/tips/88

btw, don't we need to take numLoops into account?